demo/top_down_video_demo_with_mmdet.py

```diff
@@ -83,7 +84,10 @@ def main():
 
     dataset = pose_model.cfg.data['test']['type']
 
-    cap = cv2.VideoCapture(args.video_path)
+    if args.video_path != '':
+        cap = cv2.VideoCapture(args.video_path)
+    else:
+        cap = cv2.VideoCapture(0)
```

Comment:
Is this for live camera?

Reply:
yes

Reply:
Refer to https://github.com/open-mmlab/mmaction2/blob/master/demo/webcam_demo.py for an implementation of webcam. It uses two threads for capturing and analyzing, which improves user experience

Reply:
So I suggest writing a separate file for webcam, and make this pr focus on channel order

Reply:
agree